src/backend/access/common/reloptions.c

```diff
@@ -515,22 +515,20 @@ default_reloptions(Datum reloptions, bool validate, char relkind,
 
 		compresstype = values[5];
 
```

Comment:
The code is not just for hacking here. Even we remove quicklz from system tables in the future, this code could be still kept to educate users for some time.